x-pack/plugins/features/server/oss_features.ts

```diff
@@ -71,6 +71,33 @@ export const buildOSSFeatures = ({ savedObjectTypes, includeTimelion }: BuildOSS
             },
           ],
         },
+        {
+          name: i18n.translate('xpack.features.ossFeatures.discoverSearchSessionsFeatureName', {
+            defaultMessage: 'Store Search Sessions',
+          }),
+          privilegeGroups: [
+            {
+              groupType: 'independent',
+              privileges: [
+                {
+                  id: 'store_search_session',
+                  name: i18n.translate(
+                    'xpack.features.ossFeatures.discoverStoreSearchSessionsPrivilegeName',
+                    {
+                      defaultMessage: 'Store Search Sessions',
+                    }
+                  ),
+                  includeIn: 'all',
+                  savedObject: {
+                    all: ['background-session'],
+                    read: [],
+                  },
+                  ui: ['storeSearchSession'],
+                },
+              ],
+            },
+          ],
+        },
       ],
     },
     {
```

Comment:
**question** Just want to confirm: the "Visualize" application won't need to take advantage of background searches? What about Canvas workpads?

Reply:
In the first phase, no